tests/test_ge_utils.py

```diff
@@ -78,15 +78,11 @@ def test_validate_non_dataset(file_data_asset, empty_expectation_suite):
     with pytest.raises(
         ValueError, match=r"The validate util method only supports dataset validations"
     ):
-        with pytest.warns(
-            Warning,
-            match="No great_expectations version found in configuration object.",
-        ):
```

Comment:
I don't think this has anything to do with my PR. It seems to be due to an inconsistency between `develop` and `0.18.x`. Regardless, I don't think we need the warning if we are raising an error.

Reply:
Confirmed this is failing on `0.18.x` without my changes.